cityofaustin/atd-product
# 4632 use standard footer

Fixes cityofaustin/atd-data-tech/issues/4632

Steps to test:

[Netlify preview](https://deploy-preview-48--atd-product.netlify.app/)

1. Footer should be stuck to the bottom of the main page.
2. Navigate to "Our Work", "What We Deliver", and "About Us" pages, footer should be rendering on all of them and stuck to the bottom.
3. Click on the different footer links, make sure they work.
